[rospy.client][INFO] 2017-10-02 09:52:36,332: init_node, name[/listener_11382_1506963156296], pid[11382]
[xmlrpc][INFO] 2017-10-02 09:52:36,333: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2017-10-02 09:52:36,341: Started XML-RPC server [http://ip-172-31-19-100:52912/]
[rospy.init][INFO] 2017-10-02 09:52:36,341: ROS Slave URI: [http://ip-172-31-19-100:52912/]
[rospy.impl.masterslave][INFO] 2017-10-02 09:52:36,345: _ready: http://ip-172-31-19-100:52912/
[xmlrpc][INFO] 2017-10-02 09:52:36,353: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2017-10-02 09:52:36,353: Registering with master node http://localhost:47885
[rospy.init][INFO] 2017-10-02 09:52:36,449: registered with master
[rospy.rosout][INFO] 2017-10-02 09:52:36,452: initializing /rosout core topic
[rospy.rosout][INFO] 2017-10-02 09:52:36,460: connected to core topic /rosout
[rospy.simtime][INFO] 2017-10-02 09:52:36,464: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2017-10-02 09:52:36,512: topic[/two_ints] adding connection to [http://ip-172-31-19-100:56954/], count 0
[rospy.internal][INFO] 2017-10-02 09:52:36,712: topic[/rosout] adding connection to [/rosout], count 0
[rosout][ERROR] 2017-10-02 09:52:36,843: bad callback: <function callback at 0x7f3a50a65e60>
Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/topics.py", line 720, in _invoke_callback
    cb(msg)
  File "/home/ccc_v1_w_NDU1M_86698/asn17208_Project_1/asn17209_Project_1/work/catkin_ws/src/project1_solution/scripts/solution.py", line 22, in callback
    publish(data.a+data.b)
  File "/home/ccc_v1_w_NDU1M_86698/asn17208_Project_1/asn17209_Project_1/work/catkin_ws/src/project1_solution/scripts/solution.py", line 10, in publish
    rospy.init_node('publisher', anonymous=True)
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/client.py", line 271, in init_node
    raise rospy.exceptions.ROSException("rospy.init_node() has already been called with different arguments: "+str(_init_node_args))
ROSException: rospy.init_node() has already been called with different arguments: ('listener', ['/home/ccc_v1_w_NDU1M_86698/asn17208_Project_1/asn17209_Project_1/work/catkin_ws/src/project1_solution/scripts/solution.py'], True, None, False, False)

[rospy.core][INFO] 2017-10-02 09:52:38,784: signal_shutdown [signal-2]
[rospy.internal][INFO] 2017-10-02 09:52:38,809: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2017-10-02 09:52:38,812: topic[/two_ints] removing connection to http://ip-172-31-19-100:56954/
[rospy.impl.masterslave][INFO] 2017-10-02 09:52:38,816: signal-2
[rospy.core][INFO] 2017-10-02 09:52:39,434: signal_shutdown [atexit]
